netci.groovy

```diff
@@ -59,6 +59,7 @@ build.cmd /no-deploy-extension /${configuration.toLowerCase()}
             // Build roslyn-project-system repo - we also need to set certain environment variables for building the repo with VS15 toolset.
             batchFile("""
 echo *** Build Roslyn Project System ***
```

Comment:
It would be significantly better if you moved this to be in a batch file that is checked in, rather than inline. Doing so means that you no longer have to recreate jobs for changes like this one, and that the PR has immediate validation over the changes you are making.